pubsub/topics/main.go

```diff
@@ -125,6 +147,34 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishWithSettings(client *pubsub.Client, topic string, msg []byte) error {
+	ctx := context.Background()
+	// [START publish_settings]
+	t := client.Topic(topic)
+	t.PublishSettings = pubsub.PublishSettings{
+		ByteThreshold:  5000,
+		CountThreshold: 10,
+		DelayThreshold: 100 * time.Millisecond,
+	}
+	result := t.Publish(ctx, &pubsub.Message{Data: msg})
```

Comment:
I'm not sure if this is typical for these samples, but perhaps giving some idea of what a `result` is would be nice? Like a comment or something? or use it maybe?